src/models/metadata.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const config = require('../services/config')
+
+function getMetadata () {
+  return {
+    public_key: config.getIn(['keys', 'ed25519', 'public']),
+    urls: {
```

Comment:
There are some places where it's more difficult to separate the HTTP concerns from the domain layer